tests/test_web_exceptions.py

```diff
@@ -332,3 +332,18 @@ def test_pickle(self) -> None:
             assert resp2.reason == "Zaprescheno"
             assert resp2.status == 451
             assert resp2.foo == "bar"
+
+
+class TestHTTPUnauthorized:
+    @pytest.mark.xfail(
+        raises=(TypeError, AttributeError),
```

Comment:
I'm pretty sure it raises both, no?

Reply:
Yes, on master it raises `TypeError`, but on 3.7 it raises an `AttributeError` (mentioned in issue).

In master this field is removed completely.

Reply:
Ah, yes. It's been deprecated and removed:
- https://github.com/aio-libs/aiohttp/blame/aa5cccb84a9408fb2180d8dca42da368c12ec2e1/aiohttp/web_exceptions.py#L90
- https://github.com/aio-libs/aiohttp/pull/3462/files#diff-549d85f88f5240e30a8a85dca61a84007be49b6d267f18bb85a6534a82c0f23eL85

And so this test doesn't make any sense in master. Repurpose the base branch of this PR to 3.8 instead, then.

Reply:
Did you mean 3.7? It fails on 3.7. 

It seems to work just right on 3.8

Reply:
No, we no longer add anything to the 3.7 stream but 3.8 gets updates. If it doesn't fail there — drop the xfail marker.

Reply:
My bad, it actually does fails on 3.8 too